components/Tabs/src/TabBarView/private/MDCTabBarViewItemView.m

```diff
@@ -132,4 +134,14 @@ - (CGSize)sizeThatFits:(CGSize)size {
   return CGSizeMake(width, height);
 }
 
+#pragma mark - Accessibility
+
+- (NSString *)accessibilityLabel {
+  return _accessibilityLabel ?: self.titleLabel.text;
```

Comment:
I think this would work.  What do you think?

```objc
  return [super accessibilityLabel] ?: self.titleLabel.text;
```

Reply:
Updated.